app/views/provider_interface/organisation_settings/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= content_for :browser_title, 'Organisation settings' %>
```

Comment:
We can use the translation here too (sorry :P)

Reply:
This would make sense, yes :)